turbo.json

```diff
@@ -83,27 +83,26 @@
       ]
     },
     "lint:js": {
-      "cache": false,
+      "inputs": [
+        "!{app,pages}/**/*.{md,mdx}",
+        "!*.{md,mdx}",
+        "!{app,components,layouts,pages,styles}/**/*.css"
+      ],
       "outputs": [".eslintjscache"]
     },
     "lint:md": {
-      "cache": false,
+      "inputs": ["{app,pages}/**/*.{md,mdx}", "*.{md,mdx}"],
       "outputs": [".eslintmdcache"]
     },
     "lint:css": {
-      "cache": false,
+      "inputs": ["{app,components,layouts,pages,styles}/**/*.css"],
       "outputs": [".stylelintcache"]
     },
-    "lint": {
-      "cache": false,
-      "outputs": [".eslintjscache", ".eslintmdcache", ".stylelintcache"]
-    },
```

Comment:
We don't end up needing this task because we can use the other linting tasks in parallel instead. An example of this can be found in the `package.json` changes.